spendsterIOS/Presentation/AddressScreen/Validation/PhoneNumberValidation.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  PhoneNumberValidation.swift
+//  spendsterIOS
+//
+//  Created by Dmytro Holovko on 4/16/19.
+//  Copyright © 2019 KeyToTech. All rights reserved.
+//
+
+import Foundation
+
+class PhoneNumberValidation: Validation {
+    private let phoneNumber: String
+    
+    init(phoneNumber: String) {
+        self.phoneNumber = phoneNumber
+    }
+    
+    func validate() -> Bool {
+        let format = "[+]{1}+[0-9]{12}"
```

Comment:
@Thongvor it's basically a copy-paste with Name validation. You can extract duplication into a TextMatchValidation class and pass two parameters into constructor: text, pattern